diff --git a/tests/src/Functional/DecoupledRouterFunctionalTest.php b/tests/src/Functional/DecoupledRouterFunctionalTest.php
index 654134d..5eee845 100644
--- a/tests/src/Functional/DecoupledRouterFunctionalTest.php
+++ b/tests/src/Functional/DecoupledRouterFunctionalTest.php
@@ -43,7 +43,7 @@ class DecoupledRouterFunctionalTest extends BrowserTestBase {
    *
    * @var array
    */
-  public static $modules = [
+  protected static $modules = [
     'language',
     'node',
     'path',
@@ -55,7 +55,7 @@ class DecoupledRouterFunctionalTest extends BrowserTestBase {
   /**
    * {@inheritdoc}
    */
-  protected function setUp() {
+  protected function setUp(): void {
     parent::setUp();
     $language = ConfigurableLanguage::createFromLangcode('ca');
     $language->save();
diff --git a/tests/src/Functional/DecoupledRouterInfoAlterTest.php b/tests/src/Functional/DecoupledRouterInfoAlterTest.php
index 60746e6..f13cbab 100644
--- a/tests/src/Functional/DecoupledRouterInfoAlterTest.php
+++ b/tests/src/Functional/DecoupledRouterInfoAlterTest.php
@@ -30,7 +30,7 @@ class DecoupledRouterInfoAlterTest extends BrowserTestBase {
    *
    * @var array
    */
-  public static $modules = [
+  protected static $modules = [
     'decoupled_router',
     'node',
     'path',
@@ -41,7 +41,7 @@ class DecoupledRouterInfoAlterTest extends BrowserTestBase {
   /**
    * {@inheritdoc}
    */
-  protected function setUp() {
+  protected function setUp(): void {
     parent::setUp();
     $this->drupalCreateContentType([
       'type' => 'article',
